core/finality-grandpa/src/communication/mod.rs

```diff
@@ -64,7 +64,7 @@ mod cost {
 	pub(super) const BAD_SIGNATURE: i32 = -100;
 	pub(super) const MALFORMED_COMMIT: i32 = -1000;
 	pub(super) const FUTURE_MESSAGE: i32 = -500;
-    pub(super) const UNKNOWN_VOTER: i32 = -150;
+	pub(super) const UNKNOWN_VOTER: i32 = -150;
     pub(super) const WILLFULREDUNDANT: i32 = -500;
```

Comment:
this is still spaces

Reply:
I'll address all of those in one big move at the end